src/strategies/decentraland-estate-size/index.ts

```diff
@@ -4,11 +4,20 @@ import { subgraphRequest } from '../../utils';
 export const author = '2fd';
 export const version = '0.1.0';
 
+const SUBGRAPH_QUERY_ADDRESSES_LIMIT = 1000;
```

Comment:
Even 2000 works well, but now 2500, let's try this, we can downgrade if needed :)
```suggestion
const SUBGRAPH_QUERY_ADDRESSES_LIMIT = 2000;
```